content/platform/swarm-learning/home.md

```diff
@@ -0,0 +1,66 @@
+---
+title: HPE Swarm Learning
+version: 1.2.0
+description: "HPE Swarm Learning is a decentralized, privacy-preserving Machine
+  Learning framework. "
+image: /img/platforms/guid-e80d248e-e754-498e-99d6-67508092f779-high.jpg
+width: large
+priority: 0
+frontpage: false
+active: false
+---
+HPE Swarm Learning is a decentralized, privacy-preserving Machine Learning framework. This framework utilizes the computing power at, or near, the distributed data sources to run the Machine Learning algorithms that train the models. It uses the security of a blockchain platform to share learnings with peers in a safe and secure manner. In  HPE Swarm Learning, training of the model occurs at the edge, where data is most recent, and where prompt, data-driven decisions are mostly necessary. In this completely decentralized architecture, only the insights learned are shared with the collaborating ML peers, not the raw data. This tremendously enhances data security and privacy.
```

Comment:
recommend placing "(ML)" just prior to "framework". This is the most common method of defining the abbreviation (in the first instance) prior to using it elsewhere.

Reply:
Done